plugins/browser-plugin-webview/src/index.ts

```diff
@@ -0,0 +1,76 @@
+import { BrowserPlugin, BrowserTracker, Payload } from '@snowplow/browser-tracker-core';
+import { hasMobileInterface, trackWebViewEvent } from '@snowplow/webview-tracker';
+import { Logger, SelfDescribingEvent, SelfDescribingJson } from '@snowplow/tracker-core';
+import { base64urldecode } from './utils';
+
+const _trackers: Record<string, BrowserTracker> = {};
+
+/**
+ * Forwards events to Snowplow mobile trackers running in a WebView.
+ */
+export function WebViewPlugin(): BrowserPlugin {
+  let LOG: Logger;
+
+  return {
+    activateBrowserPlugin: (tracker: BrowserTracker) => {
+      _trackers[tracker.id] = tracker;
+    },
+
+    logger: (logger: Logger) => {
+      LOG = logger;
+    },
+
+    filter: (payload) => {
+      if (hasMobileInterface() === true) {
+        LOG.debug(`Payload (event ID: ${payload.eid}) was filtered out and forwarded to WebView tracker.`);
+
+        let atomicProperties = {
+          eventName: payload.e as string,
+          trackerVersion: payload.tv as string,
+          useragent: (payload.ua as string) ?? window.navigator.userAgent,
+          url: payload.url as string | undefined,
+          title: payload.page as string | undefined,
+          referrer: payload.refr as string | undefined,
+          category: payload.se_ca as string | undefined,
+          action: payload.se_ac as string | undefined,
+          label: payload.se_la as string | undefined,
+          property: payload.se_pr as string | undefined,
+          value: payload.se_va ? parseFloat(payload.se_va as string) : undefined,
+          minXOffset: payload.pp_mix ? parseInt(payload.pp_mix as string) : undefined,
+          maxXOffset: payload.pp_max ? parseInt(payload.pp_max as string) : undefined,
+          minYOffset: payload.pp_miy ? parseInt(payload.pp_miy as string) : undefined,
+          maxYOffset: payload.pp_may ? parseInt(payload.pp_may as string) : undefined,
+        };
+        let event = getSelfDescribingEventData(payload);
+        let entities = getEntities(payload);
+        let trackers: Array<string> = Object.keys(_trackers);
+
+        trackWebViewEvent({ properties: atomicProperties, event: event, context: entities }, trackers);
```

Comment:
Can we make the last `trackers` argument optional and by default omit it?

I think it's quite unlikely (and probably not a good design) that users will use the same tracker namespace for their JS tracker and their mobile trackers. So forcing the JS tracker namespace here is probably not a good idea.

I would suggest making it configurable:
1. by default, we wouldn't specify the list of trackers so that the events will be tracked with any trackers
2. in the `WebViewPlugin` function we could add a configuration object that would let users provide the list of tracker namespaces to use.